metomi/rose/env_cat.py

```diff
@@ -25,8 +25,34 @@
 
 def main():
     """Implement "rose env-cat"."""
-    opt_parser = RoseOptionParser()
+    opt_parser = RoseOptionParser(
```

Comment:
1. For another day - It took me a long time to understand this docstring (and I'm not sure I have).
2. What behavior should I see if I just execute the command with no opts or args? Is just hanging desirable?